src/components/Reports/StaffExpenseReport/SettingsDialog/SettingsDialog.test.tsx

```diff
@@ -12,13 +16,78 @@ const TestComponent: React.FC<SettingsDialogProps> = ({
   onClose,
   selectedFilters,
 }) => (
-  <LocalizationProvider dateAdapter={AdapterLuxon}>
-    <SettingsDialog
-      isOpen={isOpen}
-      onClose={onClose}
-      selectedFilters={selectedFilters}
-    />
-  </LocalizationProvider>
+  <TestRouter>
+    <GqlMockedProvider<{ ReportsStaffExpenses: ReportsStaffExpensesQuery }>
+      mocks={{
+        ReportsStaffExpenses: {
+          reportsStaffExpenses: {
+            funds: [
+              {
+                categories: [
+                  {
+                    category: StaffExpenseCategoryEnum.Benefits,
+                    subcategories: [
+                      {
+                        breakdownByMonth: [
+                          {
+                            transactions: [
+                              {
+                                transactedAt: DateTime.now().toISO() ?? '',
```

Comment:
If we don't need `transactedAt` to be a specific value, you can do `transactions: [{}]`. That means that `transactions` will have one entry, but all of it's fields will be mocked.

```suggestion
```